.github/workflows/docker.yml

```
@@ -0,0 +1,61 @@
+name: Build Verilator Container
+
+on:
+  push:
+    tags: [ 'v*' ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

(I'd rather control when we switch to 24.04 rather than github's decision)